hugr-py/src/hugr/_dfg.py

```diff
@@ -164,6 +165,28 @@ def load(self, const: ToNode | val.Value) -> Node:
 
         return load
 
+    def call(
+        self,
+        func: ToNode,
+        *args: Wire,
+        instantiation: FunctionType | None = None,
+        type_args: list[TypeArg] | None = None,
+    ) -> Node:
+        f_op = self.hugr[func]
+        f_kind = f_op.op.port_kind(func.out(0))
+        match f_kind:
+            case FunctionKind(sig):
+                signature = sig
+            case _:
+                raise ValueError(f"Expected function type, got {f_kind}")
```

Comment:
This may be a user error, so clarify what is failing.
```suggestion
                raise ValueError(f"Expected 'func' to be a function")
```
It may be better to check the operation type and print that instead.